src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String roleName;
```

Comment:
If Role model has RoleName of type enum, use annotation @Enumerated(value = EnumType.STRING), for it to be displayed and stored properly in db.

Reply:
Role class don't use enum